### PR TITLE
Remove Gemfile.lock and adding it to .gitignore

### PR DESCRIPTION
**Pull request by vishrutshah**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: .gitignore [hunk 1]
```diff
@@ -187,4 +187,7 @@ src/generator/AutoRest.Ruby*/*/RspecTests/Generated/*
 #dnx installation
 dnx-clr-win-x86*/
 dnx-coreclr-win-x86*/
-/dnx
\ No newline at end of file
+/dnx
+
+# Gemfile.lock
+/Gemfile.lock
```
Comment (veronicagg) [inline .gitignore]: do we need the "/" at the beginning?

**Comment by veronicagg** [comment]
@vishrutshah left one comment, rest looks good, thanks for sending a PR for this! :)

File: .gitignore (hunk 1 above)
Comment (vishrutshah) [inline .gitignore]: nope, not needed.

**Comment by veronicagg** [comment]
@vishrutshah LGTM!

**Comment by vishrutshah** [comment]
Thanks for the review!!

**Pull request merged**